common-lib/src/test/java/com/google/cloud/tools/intellij/BasePluginInfoServiceTest.java

```diff
@@ -31,10 +31,10 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 
 /** Tests for {@link BasePluginInfoService}. */
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
Whoops I thought this was on a different class, nevermind. I've removed the `Silent` runner here and removed the unnecessary stubs. PTAL.